conftest.py

```diff
@@ -55,17 +51,15 @@ def pytest_addoption(parser):
     parser.addoption("--local-dir", default=None, help="Where answers are saved.")
     # Tell pytest about the local-dir option in the ini files. This
     # option is used for creating the answer directory on CI
-    parser.addini("local-dir", default="../answer-store", help="answer directory.")
+    parser.addini("local-dir",
+        default=os.path.join("..", "answer-store"),
```

Comment:
So this was rewritten following one of my comments expressing concerns regarding the Windows compatibility of this bit.
I'm now uncertain whether this implementation is indeed cross platform. There's also the issue that this will break on any platform if `pytest` isn't called from the top level of the repo.

The one way I know works everywhere to refer to parent directories would read
```python
default=str(Path(__file__).parent / "answer-store"),
```

Reply:
I think if pytest isn't called from the top-level directory then it won't find the proper `conftest.py` file, correct? Doesn't that mean that pytest must be invoked from the root yt directory?

Reply:
I'm 90% sure that pytest walks the file tree until it finds a configuration file it recognizes, so unless _we_ break this mechanism, we're supposed to be able to run pytest from anywhere if it's config file lives at the top level. This is especially useful for yt where tests files are not centralised into one location, so please don't break it :)

Note that since 6.0, pyproject.toml is supported (https://docs.pytest.org/en/stable/customize.html?highlight=configure#pyproject-toml) so we actually don't need a new file.

Reply:
I added the ini options to the `pyproject.toml` file and removed the two pytest ini files. The only option that isn't in the toml file is `--with-answer-testing`. That should be added manually on the command line when invoking pytest because that's how the answer tests are turned on and the unit and answer tests are mutually exclusive.

Reply:
Excellent. If this is already documented you can resolve this thread. I'll review more thoroughly tommorow